registry.json

```diff
@@ -14,6 +14,20 @@
     "github": "DCOneCrypto"
     },
     {
+    "ticker": "TRADE",
+    "poolId": "b99b81d78a57c8b467c5b8e976c225c95f283777ea4376a273adfe06",
+    "social": {
+      "website": "https://adatradepool.com/",
+      "twitter": "adatradepool",
```

Comment:
Twitter invalid

Reply:
Sorry for the missed copy/paste on twitter.
I updated the file.